clustered/integration-test/src/test/java/org/ehcache/clustered/sync/PassiveSyncTest.java

```diff
@@ -81,11 +84,71 @@ public void testSync() throws Exception {
 
       Cache<Long, String> cache = cacheManager.createCache("clustered-cache", config);
 
+      for (long i = -5; i < 5; i++) {
+        cache.put(i, "value" + i);
+      }
+
+      CLUSTER.getClusterControl().startOneServer();
+      CLUSTER.getClusterControl().waitForRunningPassivesInStandby();
+      CLUSTER.getClusterControl().terminateActive();
+
+      for (long i = -5; i < 5; i++) {
+        assertThat(cache.get(i), equalTo("value" + i));
+      }
+    } finally {
+      cacheManager.close();
+    }
+  }
+
+  @Ignore
```

Comment:
This test will fail due to a platform bug. Kept it here so that we can verify if the bug is fixed with the next platform release.